### PR TITLE
feat: add read more functionality to journal entries in bp

### PR DESCRIPTION
**Pull request by eddieferrer**
ACK-470

### CODE REVIEW

File: src/components/BorrowerProfile/UpdateDetails.vue [hunk 1]
```diff
@@ -57,9 +69,24 @@ export default {
...
+		},
+		truncateBody() {
+			const bodyLength = this.htmlSafeBody?.length ?? 0;
+			return bodyLength > 210;
+		},
+		shortBody() {
+			return `${this.htmlSafeBody?.substring(0, 210)}... `;
+		},
+		htmlSafeBody() {
+			return DOMPurify.sanitize(this.body, { ALLOWED_TAGS: ['b', 'i', 'em', 'strong', 'a', 'p', 'br'] });
```
Comment (mcstover) [inline src/components/BorrowerProfile/UpdateDetails.vue:89]: Would there be H_n_ header elements in here?

File: src/components/BorrowerProfile/UpdateDetails.vue [hunk 2]
```diff
@@ -57,9 +69,24 @@ export default {
...
 			return format(parseISO(this.date), 'MMMM dd, yyyy');
+		},
+		truncateBody() {
+			const bodyLength = this.htmlSafeBody?.length ?? 0;
+			return bodyLength > 210;
+		},
+		shortBody() {
+			return `${this.htmlSafeBody?.substring(0, 210)}... `;
+		},
+		htmlSafeBody() {
```
Comment (eddieferrer) [inline src/components/BorrowerProfile/UpdateDetails.vue:88]: Some journals entries have html, so better sanitize them here.

**Review by dyersituations — APPROVED** [review]
Approved pending question on header elements from @mcstover

File: src/components/BorrowerProfile/UpdateDetails.vue (hunk 1 above)
Comment (eddieferrer) [reply src/components/BorrowerProfile/UpdateDetails.vue:89]: Hmm yeah, there could possibly be. I'm not sure where this html is coming from..... I'll add h tags as allowed here as well.

**Review by mcstover — APPROVED** [review]

**Pull request merged**